ui/src/forms/editors/AutoCompleteEditor.vue

```diff
@@ -9,7 +9,12 @@
     @search="onSearch"
     @search:focus="__load"
     :clear-search-on-blur="clearOnBlur"
-  />
+  >
+    <template #option="option">
+      {{ option.label }}
+      <cite v-if="option.helptext"><br>({{ option.helptext }})</cite>
```

Comment:
`<cite>`? Interesting choice of element. Any specific reason?

Reply:
It was used  in the docs for `vue-select` 😀

Reply:
Well... let's say it's a good argument 😆